forge/build.gradle

```
@@ -49,6 +49,14 @@ shadowJar {
 
     configurations = [project.configurations.shadowCommon]
     classifier "dev-shadow"
+
+    // Replace classes with forge's version
+    exclude "dev/architectury/core/block/ArchitecturyLiquidBlock.class"
+    exclude "dev/architectury/core/fluid/ArchitecturyFlowingFluid.class"
+    exclude "dev/architectury/core/fluid/ArchitecturyFlowingFluid\$Source.class"
+    exclude "dev/architectury/core/fluid/ArchitecturyFlowingFluid\$Flowing.class"
```

```suggestion
    exclude 'dev/architectury/core/fluid/ArchitecturyFlowingFluid$Source.class'
    exclude 'dev/architectury/core/fluid/ArchitecturyFlowingFluid$Flowing.class'
```
a single-quote string doesn't need to escape $ afaik